core/issuers.js

```diff
@@ -81,6 +126,14 @@ Description must be truthy: ${description}`;
       const sinkPurse = issuer.makeEmptyPurse('sink purse');
       return sinkPurse.depositAll(srcPaymentP);
     },
+
+    getParentIssuer() {
```

Comment:
For example, `getParentIssuer` here really is the grandparent of `getChildIssuer` here, which is good.

Reply:
Good point